src/faebryk/library/has_multi_picker.py

```diff
@@ -84,3 +84,19 @@ def handle_duplicate(self, other: TraitImpl, node: Node) -> bool:
         other.pickers.extend(self.pickers)
         other.pickers.sort(key=lambda x: x[0])
         return False
+
+    @classmethod
+    def remove_if[T: Module](
+        cls, m: T, condition: Callable[[T], bool], prio: int = -10
+    ) -> Self:
+        def replace(module: Module):
+            assert module is m
```

Comment:
Those captures inhibit python from caching anonymous functions
Not really a perf bottleneck here, just wanted to note